### PR TITLE
removes bio bag compatibility w/ reproductive crossbreed because it doesn't work

### PR DESCRIPTION
**Pull request by ReddicusDragon**
# Document the changes in your pull request
you can no longer use bio bags with crossbreeds

# Changelog
:cl:  
rscdel: Removed slime crossbreed bio bag compatibility because it doesn't work  
/:cl:

### CODE REVIEW

**Comment by Moltijoe** [comment]
is it possible to make it work?

**Comment by ReddicusDragon** [comment]
> is it possible to make it work?

5 year old code, you're welcome to try

File: code/modules/research/xenobiology/crossbreeding/reproductive.dm [hunk 1]
```diff
@@ -18,15 +18,6 @@ Reproductive extracts:
 	if((last_produce + cooldown) > world.time)
 		to_chat(user, span_warning("[src] is still digesting!"))
 		return
-	if(istype(O, /obj/item/storage/bag/bio))
-		var/list/inserted = list()
-		SEND_SIGNAL(O, COMSIG_TRY_STORAGE_TAKE_TYPE, /obj/item/reagent_containers/food/snacks/monkeycube, src, 1, null, null, user, inserted)
-		if(inserted.len)
-			var/obj/item/reagent_containers/food/snacks/monkeycube/M = inserted[1]
```
Comment (ynot01) [inline code/modules/research/xenobiology/crossbreeding/reproductive.dm:25]: is it perhaps due to this line that it assumes the top item is a monkey cube? have you tried it with only monkey cubes in the bag?

Comment (Moltijoe) [reply code/modules/research/xenobiology/crossbreeding/reproductive.dm:25]: i could have sworn it's worked for me in the past, so that might be the issue

**Review by MajManatee — COMMENTED** [review]
I mean, id prefer if it was made to work, but I also have no issued with doing away with it either.

**Comment by MajManatee** [comment]
Author taking a break, feel free to re-open when you're back.

**Comment by ynot01** [comment]
if you plan on retouching on this, please make it so it finds the monkey cube instead of assuming the topmost item is a monkey cube, it's a really simple fix and we shouldn't be tossing features like this

**Pull request closed without merging**